### ISSUE TITLE
Scheduled task to bring the PSA CSV over to a local drive

### ISSUE THREAD

**Issue by hangler** [bcgov/ExitSurveyAdmin#92]
As we discussed yesterday, the plan while we await an API offering the PSA information is to retain the old ESA system for the sole purpose of copying the PSA CSV to a directory. This avoids the need to spend development effort on building SharePoint integration into the new app.

While the new app is already configured to possibly read a local CSV, the specifics of this workflow will probably require a bit of new development effort nonetheless. @robinwn let's discuss as needed.

**Comment by hangler** [comment]
After our chat this morning, @robinwn and I decided that:

1. The ExitSurveyAdmin app itself does not need to care about finding or parsing the CSV file.
2. The scheduled task will run a script (see #54) which will copy or otherwise obtain the contents of the CSV file and `POST` it as a string to the `/api/CsvExtract/EmployeesFromCsv` endpoint. The ExitSurveyAdmin app will parse this as appropriate.

**Comment by robinwn** [comment]
@hangler I'm currently testing the dev instance with one of the exported test csv's with this:
> curl -d "@EXIT_USERS.csv" -X POST -H "Content-Type: text/csv" https://<domain>/<path>/api/CsvExtract/EmployeesFromCsv

Getting a Task Long entry with a warning that contains 

> th name 'Location Group' was not found.

Headers in the src csv (EXIT_USERS>csv) are:
> EmplID,"Empl Rcd","Last Name","First Name",Birthdate,Gender,Classification,Ministry,"DeptID Desc","Job Function","Location City","Orig Hire Date","Last Day Worked",Effdt,Reason,Address1,Address2,City,Prov,"Post Code",Phone,"Appointment Status",Position,"Position Title",Age,"Leave Date",Service,"Job Code","Back Dated","Exit Count","Age Group 2","Classification Group","Service Group BCSTATS","Location Group"

csv columns (fields) have the extra double quotes removed and trimmed left & right

**Comment by hangler** [comment]
@robinwn Let me try locally. It's looking like a CSV that had been working is now failing for some reason.

**Comment by hangler** [comment]
@robinwn After some further inspection, it looks like the CsvHelper module is just not very good at processing things asynchronously — I was encountering bizarre bugs that really should not have been happening (e.g. mismatches between the column theoretically being parsed and the field in the row being loaded). For some reason those problems didn't surface beforehand.

In any case, I have switched the CSV parsing to be synchronous. However, this requires setting `AsynchronousIO` to `true` in the app configuration. I've got a working line of code that does that with Kestrel. In production, will the app still be using Kestrel? If not, I believe there is equivalent code for IIS.
